pre_commit_hooks/check_jira_pre_commit.py

```
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import argparse
+import io
+import re
+import sys
+
+COMMIT_REGEX = re.compile(r"[A-Z]+-[0-9]+")
+ERROR_MSG = "Aborting commit. Your commit message is missing a Jira Task ID, i.e. JIRA-1234."
+
+
+def parse_args(argv=None):
+    """Check commit message for Jira Task ID."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "path",
+        nargs="+",
+        help="Path to `COMMIT_EDITMSG` file.",
+    )
+
+    return parser.parse_args(argv)
+
+
+def check_commit(path: str):
+    with io.open(path, "r+") as commit_message_file:
+        commit_text = commit_message_file.read()
+
+    if commit_text.startswith("Merge "):
+        print("Ignoring git merge commit")
+        sys.exit(0)
+
+    match = re.search(COMMIT_REGEX, commit_text)
+    if not match:
+        print(ERROR_MSG)
+        sys.exit(1)
```

Can you just return a failed error retval? Like we do here - https://github.com/saritasa-nest/saritasa-pre-commit-hooks/blob/main/pre_commit_hooks/check_nginx_wide_range.py#L292

---

Used `return 1`